src/nile/commands/init.py

```diff
@@ -25,95 +26,9 @@ def init_command():
 
     # create project directories
     print("🗄  Creating project directory tree")
-    create_contracts()
-    create_tests()
-
-    with open("Makefile", "w") as fp:
-        fp.write(makefile)
+    copy_tree(os.path.join(os.path.dirname(__file__), "../base_project"), ".")
```

Comment:
Maybe using `pathlib.Path` would make this cleaner? E.g. `Path(__file__).parent / 'base_project')`

Reply:
thanks! will fix this in #26